### PR TITLE
Don't reflow PGP blocks in Mutt

### LINKED ISSUE #29: Recognize and ignore PGP blocks
This is weird and super specific, but when mutt (or gpg?) displays sig info for an email they do it in a block that looks like the following:

```
[-- Begin signature information --]
Good signature from: Gordon Fontenot (personal) <gordon@fonten.io>
                aka: keybase.io/gfontenot <gfontenot@keybase.io>
                aka: Gordon Fontenot (thoughtbot) <gordon@thoughtbot.com>
                aka: Gordon Fontenot (personal) <gordon.fontenot@me.com>
                aka: Gordon Fontenot (git) <gmoney@gitpushgitpaid.com>
            created: Thu May 12 14:26:06 2016
[-- End signature information --]
```

Unfortunately, when run through `reflow`, it looks like this on the other end:

```
[-- Begin signature information --]
Good signature from: Gordon
Fontenot (personal) <gordon@fonten.io>



 aka: keybase.io/gfontenot
<gfontenot@keybase.io>



 aka: Gordon Fontenot (thoughtbot)
<gordon@thoughtbot.com>



 aka: Gordon Fontenot (personal)
<gordon.fontenot@me.com>



 aka: Gordon Fontenot (git)
<gmoney@gitpushgitpaid.com>


  created: Thu May
12 14:26:06 2016
[-- End signature information --]
```

This is... not ideal. I think it might make sense to check for sig info blocks and just ignore them the way we do code blocks.

### PR DESCRIPTION
**Pull request by gabebw**
Fixes #29.

Please check this in Mutt to make sure it works as expected.

### CODE REVIEW

**Comment by gfontenot** [comment]
Add a test! Feel free to use the text from #29

**Comment by gabebw** [comment]
Just added a test!

**Comment by gabebw** [comment]
Just added a test!

**Comment by gabebw** [comment]
OK, _really_ added a test this time.

File: src/Reflow/Parser.hs [hunk 1]
```diff
@@ -62,6 +62,12 @@ codeBlock = do
     void eol
     return $ CodeBlock $ s <> c <> e
 
+pgpBlock :: Parser Content
+pgpBlock = fmap PGPBlock $ mappend3 <$> pgpBlockStart <*> pgpBlockContents <*> pgpBlockEnd
```
Comment (gabebw) [inline src/Reflow/Parser.hs:66]: `mappend3` could be used in the `codeBlock` method above, but that felt out of scope.

**Pull request merged**